### DISCUSSION TITLE
Model Features

### DISCUSSION THREAD

**Discussion by carlosaur** [pycaret/pycaret#1160] [Q&A] [answered]
Could you tell me how I access the "Transformed Data" dataset, after running "setup.pycaret.clustering"? I used the PCA parameter and I would like to check the new dataset  generated.

**Comment by Yard1** [comment, marked as answer]
`get_config('X')`

**Reply by carlosaur** [reply]
Thank you very much for the quick answer. Is there  a complete documentation of these functions?

**Comment by moezali1** [comment]
@carlosaur Here is the the documentation link: https://pycaret.readthedocs.io/en/latest/api/clustering.html#pycaret.clustering.get_config

**Reply by carlosaur** [reply]
Ok. Thank You all again.
